sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -326,47 +332,19 @@ function setName(entry: any, nameProperties: any): any {
  * @param response
  */
 export function buildError(errorBody: any, response: HttpOperationResponse): RestError {
-  const normalizedError: any = {};
-  const odataErrorFormat = !!errorBody["odata.error"];
   const errorProperties =
-    errorBody.Error || errorBody.error || errorBody["odata.error"] || errorBody;
+    errorBody.Error || errorBody.error || errorBody || errorBody["odata.error"];
```

Comment:
`errorBody["odata.error"]` will never get assigned to `errorProperties` because `errorBody` appears before it in the set of OR checks.

You need to update the order as

```suggestion
    errorBody.Error || errorBody.error || errorBody["odata.error"] || errorBody;
```